### PR TITLE
Some text and link additions

### PR DESCRIPTION
**Pull request by lawinslow**
@jread-usgs @ldecicco-USGS @eread-usgs thoughts?

### CODE REVIEW

**Comment by lawinslow** [comment]
P.S. I added a "contact/contribute" page.

**Comment by jread-usgs** [comment]
The pipe between contact and contribute is weird

**Comment by lawinslow** [comment]
Yeah, other ideas are appreciated.

File: about.html [hunk 1]
```diff
@@ -115,21 +115,39 @@
...
             
-            <p>Fusce ac arcu sed dolor ultrices iaculis a non est. Duis venenatis magna eget ante dignissim auctor. Nulla diam leo, semper ac arcu suscipit, pellentesque sollicitudin velit. Pellentesque fringilla massa tellus, a suscipit ligula sollicitudin eget. Vivamus in dolor scelerisque, aliquam sapien ut, tristique nisl. Nulla est nulla, bibendum vitae pulvinar eget, finibus non metus. Nunc sit amet mauris vel enim tristique molestie. Integer in turpis varius, tincidunt nisl ac, pretium ligula. Pellentesque quam ex, feugiat vel tellus eget, fringilla ornare felis. Ut malesuada at justo ornare egestas. Ut imperdiet vel arcu et finibus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Suspendisse sed gravida libero, vitae mollis massa. Suspendisse interdum, magna nec imperdiet auctor, nibh nisi porttitor purus, vel pretium metus nisl non justo.</p>
+            <p>USGS-R is a community of researchers, technicians and programmers working to build open-source science-support tools for R. 
+            Through training and development of tools in R, we are advancing how the USGS uses data, computing, and visualization.
```
Comment (jread-usgs) [inline about.html]: science-support tools *in* R

**Comment by jread-usgs** [comment]
Looks good though.

**Pull request merged**